src/test/ui/treat-err-as-bug/err.stderr

```diff
@@ -0,0 +1,23 @@
+error[E0080]: could not evaluate static initializer
+  --> $DIR/err.rs:9:21
+   |
+LL | pub static C: u32 = 0 - 1;
+   |                     ^^^^^ attempt to compute `0_u32 - 1_u32`, which would overflow
+
+thread 'rustc' panicked at 'aborting due to `-Z treat-err-as-bug=1`', compiler/rustc_errors/src/lib.rs:990:27
+
+
+error: internal compiler error: unexpected panic
+
+
+
+
+
+
+
+
+
```

Comment:
All these empty lines are because of `normalize-stderr-test "note: .*" -> ""` some notes are platform dependent and the other are not interesting. I'm not sure if there's a way to just skip the line instead of replacing it with blank.